packages/pds-ember/tests/dummy/app/components/docs/form-field/select.hbs

```diff
@@ -19,6 +19,10 @@
     <option value="Dewey">Dewey</option>
     <option value="Cheetum">Cheetum</option>
     <option value="Enhowe">Enhowe</option>
+    <option value="pangram">
+      The quick brown fox<br />
+      jumps over the lazy dog.
+    </option>
```

Comment:
This was a demonstration/attempt to try and get the option label to display on multiple lines.  However, as you can see in the screenshot, the browser ignores the `<br />` element.

![select - attempt multiline option](https://user-images.githubusercontent.com/545605/97463705-093a8a00-190e-11eb-8392-390a135a4734.png)